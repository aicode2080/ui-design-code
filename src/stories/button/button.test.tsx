import { render } from '@testing-library/react';
import { Button } from './Button';

describe('Button', () => {
  test('展示一个button组件', () => {
    const component = render(<Button label="按钮" />);
    expect(component.baseElement).toMatchSnapshot();
  });

  test('展示一个不同大button组件', () => {
    const component = render(
      <Button label="large" />,
    );
    expect(component.baseElement).toMatchSnapshot();
  });
});
