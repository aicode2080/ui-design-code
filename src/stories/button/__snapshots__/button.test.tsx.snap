// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Button > 展示一个button组件 1`] = `
<body>
  <div>
    <button
      class="storybook-button storybook-button--medium storybook-button--secondary"
      type="button"
    >
      按钮
    </button>
  </div>
</body>
`;

exports[`Button > 展示一个不同大button组件 1`] = `
<body>
  <div>
    <button
      class="storybook-button storybook-button--medium storybook-button--secondary"
      type="button"
    >
      large
    </button>
  </div>
</body>
`;
